DragDrop/DragDrop/View/BoardPage.swift

```diff
@@ -8,10 +8,12 @@
 import SwiftUI
 import SwiftData
 
-struct MainPage: View {
+struct BoardPage: View {
+    @Environment(\.modelContext) private var modelContext
     //    @Query private var tasks: [Task]
     let today = Date().formatted(.iso8601.year().month().day())
-    var tasks: [Task] = [Task(name: "one", category: "One"), Task(name: "two", category: "Two")]
+    @State var taskItems: [TaskItem] = [TaskItem(id: "1", name: "one", category: "iOS", isPinned: true, isDone: false), TaskItem(id: "2", name: "two", category: "Year", isPinned: true, isDone: false), TaskItem(id: "3", name: "three", category: "iOS", isPinned: true, isDone: false)]
```

Comment:
테스트용 더미데이터 인가요? extension 으로 빼두는게 좋아보입니다.

```suggestion
    @State var taskItems: [TaskItem] = TaskItem.fetchMockItems()
```

Reply:
평상시에 항상 저런식으로 냅다 선언해두는데 앞으로는 깔끔하게 빼두어야겠네요.